src/Cronos/CronField.cs

```diff
@@ -47,6 +47,7 @@ internal sealed class CronField
         public static readonly CronField Hours = new CronField("Hours", 0, 23, null, true);
         public static readonly CronField Minutes = new CronField("Minutes", 0, 59, null, true);
         public static readonly CronField Seconds = new CronField("Seconds", 0, 59, null, true);
+        public const int LastCommonDayOfMonth = 28;
```

Comment:
Hm, I understand why it is here, but not sure this behavior is expected – so 29, 30 and 31th days will always be wiped out. Do you know anything regarding Jenkins implementation? If it uses the same limitation, we can use it just fine. If not, I think we should consider other options of implementing this first, just to say "we did our best".